RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -69,28 +69,22 @@ PixelThresholdClusterizer::~PixelThresholdClusterizer() {}
 
 // Configuration descriptions
 void
-PixelThresholdClusterizer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
-  // siPixelClusters
-  edm::ParameterSetDescription desc;
-  desc.add<edm::InputTag>("src", edm::InputTag("siPixelDigis"));
+PixelThresholdClusterizer::fillPSetDescription(edm::ParameterSetDescription& desc) {
   desc.add<int>("ChannelThreshold", 1000);
   desc.addUntracked<bool>("MissCalibrate", true);
```

Comment:
I suppose this parameter should really be tracked. I did not include that change in this PR because the parameter is being used in the HLT menu (so would need a customize function in `customizeHLTforCMSSW`) and the change was not needed for my aim. I can, however, do that change as well if people want.

Reply:
@makortel Could you please contact HLT people and ask what they think? Thanks.

Reply:
I'd think the question is more for RECO (the HLT side is more about an implementation).

Reply:
Ok @makortel , if you ask me: yes, that parameter should be tracked. Please go ahead with the change, an provide also the HLT customization with it in this PR. Thank you!

Reply:
Thanks @perrotta, done in the last commit.